scrapers_next/ca/people.py

```diff
@@ -5,7 +5,8 @@
 
 class AssemblyList(HtmlListPage):
     source = "https://www.assembly.ca.gov/assemblymembers"
-    selector = CSS("table tbody tr", num_items=80)
+    # selector = CSS("table tbody tr", num_items=80)
+    selector = CSS("div.rt__row > div", num_items=80)
```

Comment:
Can just remove the commented out `selector`. We appreciate leaving it but if we need to see the history of a selector we can just check the history of the file on github. Also the selector shouldn't include the `> div` since we set other selectors outside of that `div` later